docs/lang/articles/performance_tuning/profiler.md

```diff
@@ -108,19 +103,26 @@ X64 Profiler(count)
 =========================================================================
 ```
 
-:::caution
-Currently the result of `KernelProfiler` could be incorrect on OpenGL backend due to its lack of support for `ti.sync()`.
+:::note
+
+- A kernel that has been fully optimized by the compiler would not generate profiling records.
+- One kernel may generate multiple records of parallel for loops because they are divided into different tasks and assigned to separate devices.
+- Currently, `kernel_profiler` only supports CPU and CUDA. Ensure that you call `ti.sync()` before performance profiling if the program is running on GPU.
+- `jit_evaluator_xxx` can be ignored because it is automatically generated by the system.
+- You are recommended to run performance profiling several times to observe the minimum or average execution time.
```

Comment:
"You are recommended to do" is not grammatical. Use either of the following:
- Taichi recommends doing xxx
- It is recommended that you do